jest.conf.js

```diff
@@ -12,4 +12,6 @@ module.exports = Object.assign(baseConfig, {
     'packages/hashi/src/*.js',
     'packages/kolibri-tools/lib/src/*.js',
   ],
+  maxConcurrency: 2,
```

Comment:
What is the reason for this? If the tests are having issues being parallelized, that tends to indicate there's an issue in the tests themselves (this only affects running on developer machines, as in a CI environment, Jest automatically runs tests sequentially instead).

Reply:
Noooo I thought I deleted it, Im sorry. I changed it bc it was taking all my processors and my computer died trying to run the tests.

Reply:
No worries - I did wonder if that was the case!

In future, you can also just pass additional jest CLI options: https://jestjs.io/docs/cli#--maxconcurrencynum for maxConcurrency and maxWorkers to the test invocation in Kolibri, it _should_ pass through the additional arguments.

Reply:
Yes, for some reason I thought CLI options wasn't working but I think it was because I was passing maxConcurrency instead of maxWorkers. Thanks!

Reply:
Good to know, I had to do a bit of work to make sure our thin wrapper around jest passed through the additional CLI arguments, so I am glad that's still working!